exporter/awskinesisexporter/README.md

```diff
@@ -15,6 +15,9 @@ The following settings can be optionally configured:
     - `kinesis_endpoint` (no default)
     - `region` (default = us-west-2): the region that the kinesis stream is deployed in
     - `role` (no default): The role to be used in order to send data to the kinesis stream
+- `encoding`
+    - `name` (default = jaeger): defines the export type to be used to send to kinesis (available is `otlp-proto`, `otlp-json`, `zipkin-proto`, `zipkin-json`, `jaeger`)
```

Comment:
Since this is otel collector, I think we do need to eventually change this default to `otlp-proto`. Any idea what a good time to change it is?

Reply:
That is a good question, I have intentionally left it since that was the behaviour before I started updating things.

However, now would be the best time to change it.